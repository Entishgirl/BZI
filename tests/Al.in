&CONTROL
    pseudo_dir = '/fslhome/jeremy12/quantum_espresso/integration_convergence/Al'
    calculation = 'scf'
    prefix = 'Al'
/

&SYSTEM
    nat = 4
    ntyp = 1
    ibrav = 2
    ecutwfc = 40.0
    occupations ='smearing'
    celldm(1) = 4.046
    smearing = "gaussian"
    degauss = 0.000100001
/

&ELECTRONS
    mixing_beta = 0.7
    conv_thr = 1.0d-8
/

ATOMIC_SPECIES
 Al 26.982 Al.pbe-high.UPF

ATOMIC_POSITIONS angstrom
 Al 0.00000000 0.00000000 0.00000000
 Al 2.02300000 2.02300000 0.00000000
 Al 0.00000000 2.02300000 2.02300000
 Al 2.02300000 0.00000000 2.02300000

K_POINTS automatic
 2 2 2 0 0 0
