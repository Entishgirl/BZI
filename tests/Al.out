
     Program PWSCF v.6.1 (svn rev. 13369) starts on  2Oct2017 at 22: 7: 6 

     This program is part of the open-source Quantum ESPRESSO suite
     for quantum simulation of materials; please cite
         "P. Giannozzi et al., J. Phys.:Condens. Matter 21 395502 (2009);
          URL http://www.quantum-espresso.org", 
     in publications or presentations arising from this work. More details at
     http://www.quantum-espresso.org/quote

     Parallel version (MPI), running on     1 processors
     Waiting for input...
     Reading input from standard input

     Current dimensions of program PWSCF are:
     Max number of different atomic species (ntypx) = 10
     Max number of k-points (npk) =  40000
     Max angular momentum in pseudopotentials (lmaxx) =  3

     Subspace diagonalization in iterative solution of the eigenvalue problem:
     a serial algorithm will be used


     G-vector sticks info
     --------------------
     sticks:   dense  smooth     PW     G-vecs:    dense   smooth      PW
     Sum          85      85     37                  537      537     137



     bravais-lattice index     =            2
     lattice parameter (alat)  =       4.0460  a.u.
     unit-cell volume          =      16.5584 (a.u.)^3
     number of atoms/cell      =            4
     number of atomic types    =            1
     number of electrons       =        12.00
     number of Kohn-Sham states=           10
     kinetic-energy cutoff     =      40.0000  Ry
     charge density cutoff     =     160.0000  Ry
     convergence threshold     =      1.0E-08
     mixing beta               =       0.7000
     number of iterations used =          8  plain      mixing
     Exchange-correlation      = SLA-PW-PBX-PBC ( 1  4  3  4 0 0)

     celldm(1)=   4.046000  celldm(2)=   0.000000  celldm(3)=   0.000000
     celldm(4)=   0.000000  celldm(5)=   0.000000  celldm(6)=   0.000000

     crystal axes: (cart. coord. in units of alat)
               a(1) = (  -0.500000   0.000000   0.500000 )  
               a(2) = (   0.000000   0.500000   0.500000 )  
               a(3) = (  -0.500000   0.500000   0.000000 )  

     reciprocal axes: (cart. coord. in units 2 pi/alat)
               b(1) = ( -1.000000 -1.000000  1.000000 )  
               b(2) = (  1.000000  1.000000  1.000000 )  
               b(3) = ( -1.000000  1.000000 -1.000000 )  


     PseudoPot. # 1 for Al read from file:
     /fslhome/jeremy12/quantum_espresso/integration_convergence/Al/Al.pbe-high.UPF
     MD5 check sum: 67d8cc2cb872b12605a83b14c98247d0
     Pseudo is Norm-conserving, Zval =  3.0
     Generated in analytical, separable form
     Using radial grid of 1135 points,  3 beta functions with: 
                l(1) =   0
                l(2) =   0
                l(3) =   1

     atomic species   valence    mass     pseudopotential
        Al             3.00    26.98200     Al( 1.00)

      6 Sym. Ops. (no inversion) found



   Cartesian axes

     site n.     atom                  positions (alat units)
         1           Al  tau(   1) = (   0.0000000   0.0000000   0.0000000  )
         2           Al  tau(   2) = (   0.9448631   0.9448631   0.0000000  )
         3           Al  tau(   3) = (   0.0000000   0.9448631   0.9448631  )
         4           Al  tau(   4) = (   0.9448631   0.0000000   0.9448631  )

     number of k points=     4 (tetrahedron method)
                       cart. coord. in units 2pi/alat
        k(    1) = (   0.0000000   0.0000000   0.0000000), wk =   0.2500000
        k(    2) = (   0.5000000  -0.5000000   0.5000000), wk =   0.7500000
        k(    3) = (   0.0000000  -1.0000000   0.0000000), wk =   0.7500000
        k(    4) = (   0.5000000   0.5000000   0.5000000), wk =   0.2500000

     Dense  grid:      537 G-vectors     FFT dimensions: (  12,  12,  12)

     Estimated max dynamical RAM per process >       0.50MB

     Initial potential from superposition of free atoms

     starting charge   11.99084, renormalised to   12.00000
     Starting wfc are   16 randomized atomic wfcs

     total cpu time spent up to now is        0.2 secs

     per-process dynamical memory:     3.5 Mb

     Self-consistent Calculation

     iteration #  1     ecut=    40.00 Ry     beta=0.70
     Davidson diagonalization with overlap
     ethr =  1.00E-02,  avg # of iterations =  5.8

     total cpu time spent up to now is        0.2 secs

     total energy              =     174.49755335 Ry
     Harris-Foulkes estimate   =     141.71362849 Ry
     estimated scf accuracy    <      45.63212997 Ry

     iteration #  2     ecut=    40.00 Ry     beta=0.70
     Davidson diagonalization with overlap
     ethr =  1.00E-02,  avg # of iterations =  2.2

     total cpu time spent up to now is        0.2 secs

     total energy              =     172.75049699 Ry
     Harris-Foulkes estimate   =     171.70338738 Ry
     estimated scf accuracy    <       1.35350084 Ry

     iteration #  3     ecut=    40.00 Ry     beta=0.70
     Davidson diagonalization with overlap
     ethr =  1.00E-02,  avg # of iterations =  2.2

     total cpu time spent up to now is        0.2 secs

     total energy              =     172.57592912 Ry
     Harris-Foulkes estimate   =     172.57528945 Ry
     estimated scf accuracy    <       0.00990234 Ry

     iteration #  4     ecut=    40.00 Ry     beta=0.70
     Davidson diagonalization with overlap
     ethr =  8.25E-05,  avg # of iterations =  2.0

     total cpu time spent up to now is        0.3 secs

     total energy              =     172.57423718 Ry
     Harris-Foulkes estimate   =     172.57446005 Ry
     estimated scf accuracy    <       0.00321822 Ry

     iteration #  5     ecut=    40.00 Ry     beta=0.70
     Davidson diagonalization with overlap
     ethr =  2.68E-05,  avg # of iterations =  1.0

     total cpu time spent up to now is        0.3 secs

     total energy              =     172.57404859 Ry
     Harris-Foulkes estimate   =     172.57412698 Ry
     estimated scf accuracy    <       0.00105640 Ry

     iteration #  6     ecut=    40.00 Ry     beta=0.70
     Davidson diagonalization with overlap
     ethr =  8.80E-06,  avg # of iterations =  1.0

     total cpu time spent up to now is        0.3 secs

     total energy              =     172.57399496 Ry
     Harris-Foulkes estimate   =     172.57402981 Ry
     estimated scf accuracy    <       0.00030765 Ry

     iteration #  7     ecut=    40.00 Ry     beta=0.70
     Davidson diagonalization with overlap
     ethr =  2.56E-06,  avg # of iterations =  4.5

     total cpu time spent up to now is        0.3 secs

     total energy              =     172.57397688 Ry
     Harris-Foulkes estimate   =     172.57397708 Ry
     estimated scf accuracy    <       0.00000156 Ry

     iteration #  8     ecut=    40.00 Ry     beta=0.70
     Davidson diagonalization with overlap
     ethr =  1.30E-08,  avg # of iterations =  1.8

     total cpu time spent up to now is        0.3 secs

     total energy              =     172.57397670 Ry
     Harris-Foulkes estimate   =     172.57397669 Ry
     estimated scf accuracy    <       0.00000003 Ry

     iteration #  9     ecut=    40.00 Ry     beta=0.70
     Davidson diagonalization with overlap
     ethr =  2.69E-10,  avg # of iterations =  1.8

     total cpu time spent up to now is        0.3 secs

     total energy              =     172.57397669 Ry
     Harris-Foulkes estimate   =     172.57397669 Ry
     estimated scf accuracy    <       0.00000001 Ry

     iteration # 10     ecut=    40.00 Ry     beta=0.70
     Davidson diagonalization with overlap
     ethr =  9.29E-11,  avg # of iterations =  1.0

     total cpu time spent up to now is        0.3 secs

     End of self-consistent calculation

          k = 0.0000 0.0000 0.0000 (    65 PWs)   bands (ev):

     8.4972  13.4207  13.6719  13.6719  37.6953  37.6953  37.7436  43.5118
    43.5118  84.4353

          k = 0.5000-0.5000 0.5000 (    70 PWs)   bands (ev):

    -6.3826   4.6733   7.5091   7.7205  35.8214  35.9668  39.5326  48.0515
    48.1570  80.9232

          k = 0.0000-1.0000 0.0000 (    64 PWs)   bands (ev):

     3.2720  12.9556  13.1129  15.8590  27.2614  31.5269  56.3308  56.7839
    56.7942 117.8606

          k = 0.5000 0.5000 0.5000 (    70 PWs)   bands (ev):

    -6.6101   4.7046   7.7376   7.7376  35.9858  35.9858  39.5725  47.9988
    47.9988  81.0608

     the Fermi energy is    37.6961 ev

!    total energy              =     172.57397669 Ry
     Harris-Foulkes estimate   =     172.57397669 Ry
     estimated scf accuracy    <          1.0E-09 Ry

     The total energy is the sum of the following terms:

     one-electron contribution =     -13.72740223 Ry
     hartree contribution      =      29.48566464 Ry
     xc contribution           =     -23.69388655 Ry
     ewald contribution        =     180.50960083 Ry

     convergence has been achieved in  10 iterations

     Writing output data file Al.save

     init_run     :      0.13s CPU      0.14s WALL (       1 calls)
     electrons    :      0.12s CPU      0.13s WALL (       1 calls)

     Called by init_run:
     wfcinit      :      0.01s CPU      0.01s WALL (       1 calls)
     potinit      :      0.00s CPU      0.00s WALL (       1 calls)

     Called by electrons:
     c_bands      :      0.08s CPU      0.09s WALL (      10 calls)
     sum_band     :      0.01s CPU      0.02s WALL (      10 calls)
     v_of_rho     :      0.02s CPU      0.02s WALL (      11 calls)
     mix_rho      :      0.00s CPU      0.00s WALL (      10 calls)

     Called by c_bands:
     init_us_2    :      0.00s CPU      0.00s WALL (      84 calls)
     cegterg      :      0.08s CPU      0.09s WALL (      40 calls)

     Called by sum_band:

     Called by *egterg:
     h_psi        :      0.05s CPU      0.06s WALL (     137 calls)
     g_psi        :      0.00s CPU      0.00s WALL (      93 calls)
     cdiaghg      :      0.02s CPU      0.03s WALL (     133 calls)

     Called by h_psi:
     h_psi:pot    :      0.05s CPU      0.06s WALL (     137 calls)
     h_psi:calbec :      0.00s CPU      0.00s WALL (     137 calls)
     vloc_psi     :      0.05s CPU      0.05s WALL (     137 calls)
     add_vuspsi   :      0.00s CPU      0.00s WALL (     137 calls)

     General routines
     calbec       :      0.00s CPU      0.00s WALL (     137 calls)
     fft          :      0.00s CPU      0.00s WALL (     120 calls)
     fftw         :      0.05s CPU      0.06s WALL (    2558 calls)
     davcio       :      0.00s CPU      0.00s WALL (       4 calls)

     Parallel routines
     fft_scatter  :      0.00s CPU      0.00s WALL (    2678 calls)

     PWSCF        :     0.36s CPU         0.41s WALL


   This run was terminated on:  22: 7: 6   2Oct2017            

=------------------------------------------------------------------------------=
   JOB DONE.
=------------------------------------------------------------------------------=
